sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/models/CommunicationRoom.java

```diff
@@ -13,28 +12,21 @@ public final class CommunicationRoom {
     private final String roomId;
     private final OffsetDateTime validFrom;
     private final OffsetDateTime validUntil;
-    private final OffsetDateTime createdTime;
-    private final RoomJoinPolicy roomJoinPolicy;
-    private final List<RoomParticipant> participants;
+    private final OffsetDateTime createdAt;
 
     /**
      * The default constructor of CommunicationRoom.
      *
      * @param roomId The Room Id.
      * @param validFrom The starting time point of the room.
```

Comment:
lets have better descriptions for these properties. Can we copy the generated descriptions from Room Model: 

  /*
     * Unique identifier of a room. This id is server generated.
     */
    @JsonProperty(value = "id", required = true)
    private String id;

    /*
     * The timestamp when the room was created at the server. The timestamp is in RFC3339 format:
     * `yyyy-MM-ddTHH:mm:ssZ`.
     */
    @JsonProperty(value = "createdAt", required = true)
    private OffsetDateTime createdAt;

    /*
     * The timestamp from when the room is open for joining. The timestamp is in RFC3339 format:
     * `yyyy-MM-ddTHH:mm:ssZ`.
     */
    @JsonProperty(value = "validFrom", required = true)
    private OffsetDateTime validFrom;

    /*
     * The timestamp from when the room can no longer be joined. The timestamp is in RFC3339 format:
     * `yyyy-MM-ddTHH:mm:ssZ`.
     */
    @JsonProperty(value = "validUntil", required = true)
    private OffsetDateTime validUntil;